src/main/resources/application.properties

```diff
@@ -8,4 +8,9 @@ quarkus.qute.suffixes=qute.html,qute.txt,html,txt,graphql
 quarkus.kubernetes-client.trust-certs=true
 quarkus.openshift.labels."app"=gh-activity
 quarkus.openshift.route.expose=true
-quarkus.openshift.env.secrets=gh-activity-token
\ No newline at end of file
+quarkus.openshift.env.secrets=gh-activity-token
+
+activity.daily-status.repositories.beefy-scenarios=https://github.com/quarkus-qe/beefy-scenarios/actions/workflows/daily.yaml
+activity.daily-status.repositories.quarkus-test-framework=https://github.com/quarkus-qe/quarkus-test-framework/actions/workflows/daily.yaml
+activity.daily-status.repositories.quarkus-test-suite=https://github.com/quarkus-qe/quarkus-test-suite/actions/workflows/daily.yaml
+activity.daily-status.repositories.quarkus-extensions-combinations=https://github.com/quarkus-qe/quarkus-extensions-combinations/actions/workflows/daily.yml
```

Comment:
We should have daily runs for StartStopTS too (followup action)

Reply:
I've just realised that the same action for PR is configured to be run daily, so I've added the startstop ts too here.